src/client/datascience/editor-integration/codewatcher.ts

```diff
@@ -1029,7 +1021,7 @@ export class CodeWatcher implements ICodeWatcher {
         if (!result) {
             // If our cell result was a failure (but not an exception) show an error
             // for the count of cells cancelled
-            await this.addErrorMessage(file, leftCount);
+            await this.addErrorMessage(interactiveWindow, leftCount);
 
             // Throw to break out of the promise chain
             throw new InteractiveCellResultError();
```

Comment:
This isn't actually working. Need to move it into the IW for it to break the chain. More updates coming ...